x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_define_rule/index.tsx

```diff
@@ -59,177 +59,178 @@ const getStepDefaultValue = (
-      setOpenTimelineSearch(true);
-    }, []);
-
-    const handleCloseTimelineSearch = useCallback(() => {
-      setOpenTimelineSearch(false);
-    }, []);
-
-    return isReadOnlyView && myStepData != null ? (
-      <StepRuleDescription
-        direction={descriptionDirection}
-        indexPatterns={indexPatternQueryBar as IIndexPattern}
-        schema={schema}
-        data={myStepData}
-      />
-    ) : (
-      <>
-        <Form form={form} data-test-subj="stepDefineRule">
-          <CommonUseField
-            path="index"
-            config={{
-              ...schema.index,
-              labelAppend: !localUseIndicesConfig ? (
-                <EuiButtonEmpty size="xs" onClick={handleResetIndices}>
-                  <small>{i18n.RESET_DEFAULT_INDEX}</small>
-                </EuiButtonEmpty>
-              ) : null,
-            }}
-            componentProps={{
-              idAria: 'detectionEngineStepDefineRuleIndices',
-              'data-test-subj': 'detectionEngineStepDefineRuleIndices',
-              euiFieldProps: {
-                fullWidth: true,
-                isDisabled: isLoading,
-                placeholder: '',
-              },
-            }}
-          />
-          <UseField
-            path="queryBar"
-            config={{
-              ...schema.queryBar,
-              labelAppend: (
-                <EuiButtonEmpty size="xs" onClick={handleOpenTimelineSearch}>
-                  <small>{i18n.IMPORT_TIMELINE_QUERY}</small>
-                </EuiButtonEmpty>
-              ),
-            }}
-            component={QueryBarDefineRule}
-            componentProps={{
-              browserFields,
-              loading: indexPatternLoadingQueryBar,
-              idAria: 'detectionEngineStepDefineRuleQueryBar',
-              indexPattern: indexPatternQueryBar,
+  useEffect(() => {
+    if (setForm != null) {
+      setForm(RuleStep.defineRule, form);
+    }
+  }, [form]);
+
+  const handleResetIndices = useCallback(() => {
+    const indexField = form.getFields().index;
+    indexField.setValue(indicesConfig);
+  }, [form, indicesConfig]);
+
+  const handleOpenTimelineSearch = useCallback(() => {
+    setOpenTimelineSearch(true);
+  }, []);
+
+  const handleCloseTimelineSearch = useCallback(() => {
+    setOpenTimelineSearch(false);
+  }, []);
+
+  return isReadOnlyView && myStepData != null ? (
+    <StepRuleDescription
+      direction={descriptionDirection}
+      indexPatterns={indexPatternQueryBar as IIndexPattern}
+      schema={schema}
+      data={myStepData}
+    />
+  ) : (
+    <>
+      <Form form={form} data-test-subj="stepDefineRule">
+        <CommonUseField
+          path="index"
+          config={{
+            ...schema.index,
+            labelAppend: !localUseIndicesConfig ? (
+              <EuiButtonEmpty
+                size="xs"
+                onClick={handleResetIndices}
+                flush="right"
+                iconType="refresh"
```

Comment:
Nit: Any way we can make the icon 14x14px, like the designs?

Reply:
it's a standard icon size configured by EUI

Reply:
![Screenshot 2020-01-13 at 11 55 41](https://user-images.githubusercontent.com/5188868/72250343-aa648800-35fb-11ea-9d7b-eb0831077593.png)